### PR TITLE
[Docs](bangc-ops): Update accuracy standard.

### PR DESCRIPTION
**Pull request by Unireverse**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Update accuracy standard docs.

## 2. Modification

modified:   docs/MLU-OPS-Accuracy-Acceptance-Standard.md

## 3. Test Report

Update docs. No test.

### CODE REVIEW

File: docs/MLU-OPS-Accuracy-Acceptance-Standard.md [hunk 1]
```diff
@@ -6,34 +6,95 @@ MLU-OPS 算子精度验收标准
...
 
-<img src="./Accuracy-Standard-Formula.png" alt="img" style="zoom:60%;" />
+$evaluated\\_data$ 表示待评价数据， $baseline\\_data$ 表示基准数据。通常情况下， $baseline\\_data$ 为 CPU 计算结果或其他计算设备计算结果。
 
-其中：
+**$diff1$：平均相对误差**
 
-- evaluated_data 表示待评价数据，baseline_data 表示基准数据。通常情况下baseline_data为cpu计算结果。
+$$
+diff1 = \frac{ \sum |evalutated \\_ data - baseline \\_ data|}{\sum  |baseline \\_ data| }
```
Comment (PetrelYy) [inline docs/MLU-OPS-Accuracy-Acceptance-Standard.md:23]: evaluated 拼错了

File: docs/MLU-OPS-Accuracy-Acceptance-Standard.md [hunk 2]
```diff
@@ -6,34 +6,95 @@ MLU-OPS 算子精度验收标准
...
+
+**$diff3_2$：最大单点绝对误差**
+
+$$
+diff3_2 = \mathop{max}\limits_{i}|evalutated \\_ data_i - baseline \\_ data_i|
+$$
+
+**$diff3$：综合单点误差**
+
+$diff3 = (m_1, m_2)$ 。根据阈值 $th$ 将比较数据 $P$ 划分为两部分 $P_1$ 、 $P_2$ ，分别计算最大相对单点误差 $m_1$ 与最大绝对单点误差 $m_2$ 。 $P_1$ , $P_2$ , $m_1$, $m_2$ 计算方式如下：
```
Comment (PetrelYy) [inline docs/MLU-OPS-Accuracy-Acceptance-Standard.md:46]: 最大相对单点误差、最大单点相对误差  表述没一致

File: docs/MLU-OPS-Accuracy-Acceptance-Standard.md [hunk 3]
```diff
@@ -6,34 +6,95 @@ MLU-OPS 算子精度验收标准
...
 | :-: | :----:      | :-----------------------------------------------------------------: | :--------------------------------: |
-|  1  | 卷积类      | 使用了卷积指令的算子 | float: diff1 <= 1e-5 && diff2 <= 1e-5<br />half: diff1 <= 3e-3 && diff2 <= 3e-3   |
-|  2  | 累加类      | 规约求和类算子。比如 reducesum、reducemean。结果与累加顺序有关。    | diff1 <= 3e-3 && diff2 <= 3e-3     |
-|  3  | 激活类      | 激活类算子是指算子的实现用到了线性插值、泰勒展开。                  |diff1 <= 3e-3 && diff2<=3e-3        |      
-|  4  | 算术类      | 例如加减乘除。                                                      |             diff3 ==0              |
-|  5  | 纯 IO       | 纯 IO 类的算子，不涉及任何运算。比如：concat、split。               |         diff3 ==0                  |
-|  9  | 复合类      | 如果算子是由上面几种类型算子中的一种或几种复合组成，则算做复合算子。|diff1 <= 3e-3 && diff2 <= 3e-3      |
-| 10  |atomicAdd 类 | 使用了 atomic 指令的算子，多次运行结果可能不同。                    | diff1<=3e-3 && diff2 <= 3e-3       |
+|  1  | 卷积类      | 使用了卷积指令的算子 | float: $diff1 \leq 1e^{-5}$ && $diff2 \leq 1e^{-5}$<br />half: $diff1 \leq 3e^{-3}$ && $diff2 \leq 3e^{-3}$   |
+|  2  | 累加类      | 规约求和类算子。比如 reducesum、reducemean。结果与累加顺序有关。    | $diff1 \leq 3e^{-3}$ && $diff2 \leq 3e^{-3}$     |
```
Comment (PetrelYy) [inline docs/MLU-OPS-Accuracy-Acceptance-Standard.md:95]: 上文表述是 fp32 fp16，统一下吧

**Review by duzekunKTH — APPROVED** [review]

**Review by mahxn0 — APPROVED** [review]

File: docs/MLU-OPS-Accuracy-Acceptance-Standard.md (hunk 1 above)
Comment (Unireverse) [reply docs/MLU-OPS-Accuracy-Acceptance-Standard.md]: done

**Review by mahxn0 — APPROVED** [review]

**Review by PetrelYy — APPROVED** [review]

**Pull request merged**